test/NameBinding/Inputs/NamedLazyMembers/NamedLazyMembers.h

```diff
@@ -0,0 +1,33 @@
+#import <Foundation/Foundation.h>
+
+@protocol doer
+
+- (void) doSomeWork;
```

Comment:
Test style nitpick 2: No space after the close paren.

Reply:
Fixed.